files/default/distro

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
```

Comment:
Why not creating the file directly with chef since you have access the distribution definition?

Reply:
Because it is unnecessary, as this one is working.

Reply:
I don't understand we only support CentOS7 so we only have to handle CentOS 7.

Reply:
it works on debian/ubuntu also